src/PerspectivePanel.tsx

```diff
@@ -1,17 +1,23 @@
 import { Alert } from '@grafana/ui';
 import { config } from '@grafana/runtime';
+import { grafanaDataFrameToArrowTable } from '@grafana/data';
 import { HTMLPerspectiveViewerElement } from '@finos/perspective-viewer';
 import { Props } from 'types';
-import { toPerspectiveData } from './transformers';
 import React, { createRef, PureComponent } from 'react';
 import '@finos/perspective-viewer';
 import '@finos/perspective-viewer/themes/all-themes.css';
 import '@finos/perspective-viewer-d3fc';
 import '@finos/perspective-viewer-hypergrid';
 
-export class PerspectivePanel extends PureComponent<Props> {
+interface State {
+  hasNotice: boolean;
```

Comment:
since "hasNotice" is a little weird... maybe we can/should remove it an just use `this.props.data.length > 1` as the flag for if we should show data or not?

Reply:
Renamed to `ignoredSeries` instead.